contract_variable_quantity/models/abstract_contract_line.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2016 Tecnativa - Pedro M. Baeza
+# Copyright 2018 Tecnativa - Carlos Dauden
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountAbstractAnalyticContractLine(models.AbstractModel):
+    _inherit = 'account.abstract.analytic.contract.line'
```

Comment:
Shoudn't it be     _inherit = 'contract.abstract.contract.line'
 ?

Reply:
@hparfr yes, the disconnect from analytics in #207 is fairly recent. It is now complete and needs to be merged then we'll update related PRs.

Reply:
I'm trying to review it this week.

Reply:
I did update it quickly https://github.com/sbejaoui/contract/pull/15